IdentityServer/v7/docs/content/ui/server_side_sessions/session_management.md

```diff
@@ -19,6 +19,10 @@ The [session management service]({{<ref "/reference/services/session_management_
 
 The Quickstart UI contains a simple administrative page (under the "ServerSideSessions" folder) that uses the *ISessionManagementService* API.
 
+{{% notice note %}}
+In the template the administrative page is protected by checking if the caller uses a local IP address. We strongly recommend to add authorization suitable to your organization for non-test environments by adding an authorization policy and enforce that using the *Authorize* attribute for example.
```

Comment:
Local IP address isn't true yet, because we haven't released the templates with that update.

```suggestion
The Quickstart session administrative page requires a logged in user to manage sessions. We strongly recommend that you add additional authorization suitable to your organization by adding an authorization policy.
```